### PR TITLE
enhance: support default value in import

### PR DESCRIPTION
**Pull request by smellthemoon**
https://github.com/milvus-io/milvus/issues/31728

### CODE REVIEW

**Comment by smellthemoon** [comment]
rerun ut

File: internal/util/importutilv2/parquet/field_reader.go [hunk 1]
```diff
@@ -212,6 +216,173 @@ func ReadBoolData(pcr *FieldReader, count int64) (any, error) {
 	return data, nil
 }
 
+func fillWithDefaultValue(c *FieldReader, count int64) (any, []bool, error) {
```
Comment (bigsheeper) [inline internal/util/importutilv2/parquet/field_reader.go:219]: Can we merge this function into `FieldReader.Next()` func to reduce dup codes?

**Review by congqixia — APPROVED** [review]
/approve

**Pull request merged**